virtual_rainforest/entry_points.py

```diff
@@ -68,7 +68,7 @@ def _parse_command_line_params(
         raise to_raise
 
 
-def _vr_run_cli() -> None:
+def vr_run_cli() -> None:
```

Comment:
Why is this function renamed? I will admit that I don't really no what was implied by it name starting with `_` in the first place?

Reply:
The underscore implies it's private to `entry_points.py`, which it isn't. Even before I wrote that test, it was still being invoked by the poetry's command-line wrapper tool.

This does mean you might need to rerun `poetry install` in order to get a working `vr_run`.